### DISCUSSION TITLE
Update Admin Global DUO fields to match DUO

### DISCUSSION THREAD

**Discussion by Gerardv514** [dani-garcia/vaultwarden#5135] [General]
In DUO they have Client ID, Client Secret, and API Hostname. In a user's personal vault, the DUO fields also have Client ID, Client Secret, and API Hostname.

If you go into VW admin panel, Global DUO Settings the field names do not match. Currently the fields are labeled as: Integration Key, Secret Key, and Host. Are these named this way for a reason or can these be renamed so they can match?

Screenshots are below.

### **DUO Website, Protect an App:**
![image](https://github.com/user-attachments/assets/7da62570-a1eb-4d1a-b6d5-54d33e6aca0c)

### **Vaultwarden Personal Account DUO Settings**
![image](https://github.com/user-attachments/assets/a781b27c-00db-4e7e-8609-146e94054cf5)

### **Vaultwarden Admin Panel Global DUO Settings**
![image](https://github.com/user-attachments/assets/cdd608e3-f9ef-4738-bc84-b38ce4afed38)

**Comment by BlackDex** [comment]
If I'm correct, DUO Changed those names on there side. While it might be a bit confusing, it should still be more or less clear.
- Client ID > Integeration Key
- Client Secret > Secret Key
- API hostname > Host

**Reply by Gerardv514** [reply]
Is it possible to protect the admin page via duo push?

**Reply by BlackDex** [reply]
Probably, same as TOTP or Webauthn of course. But it takes time to add those. And it might not have that huge of effect.

**Comment by Gerardv514** [comment]
@0x0fbc do you think this is something within your expertise?
